sdk/python/kfp/components/executor_types.py

```diff
@@ -0,0 +1,92 @@
+import json
+import inspect
+from typing import Callable, Dict
+from kfp.components._python_op import RuntimeArtifact, InputArtifact, OutputArtifact, InputPath, OutputPath
+
+
+class Executor():
```

Comment:
You've previously added class `Executor` in `executor.py`. Can we explain the main motivation for having two classes with same names to the user?

Reply:
Old file, accidentally committed. Removed.

Reply:
//You might want to push the latest changes.

Reply:
Done.